go.mod

```diff
@@ -20,5 +21,31 @@ require (
 	k8s.io/client-go v0.17.1
 	k8s.io/component-base v0.17.0
 	k8s.io/klog v1.0.0
+	k8s.io/kubernetes v1.17.2
+	monis.app/go v0.0.0-20190702030534-c65526068664 // indirect
 	sigs.k8s.io/controller-runtime v0.4.0
 )
+
+replace (
+	k8s.io/api => k8s.io/api v0.17.0
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.0
+	k8s.io/apimachinery => k8s.io/apimachinery v0.17.0
+	k8s.io/apiserver => k8s.io/apiserver v0.17.0
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.17.0
+	k8s.io/client-go => k8s.io/client-go v0.17.0
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.0
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.17.0
+	k8s.io/code-generator => k8s.io/code-generator v0.17.0
+	k8s.io/component-base => k8s.io/component-base v0.17.0
+	k8s.io/cri-api => k8s.io/cri-api v0.17.0
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.17.0
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.17.0
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.17.0
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.17.0
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.17.0
+	k8s.io/kubectl => k8s.io/kubectl v0.17.0
+	k8s.io/kubelet => k8s.io/kubelet v0.17.0
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.0
+	k8s.io/metrics => k8s.io/metrics v0.17.0
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.0
+)
```

Comment:
Shouldn't this change be part of the "Update deps" commit instead?

Also, will we "backport" this patch to 4.4? (= cherry-pick to`release-4.4` branch)?

IMO we should have e2e tests there, but this is a rather big change (1.200+ files updated).backport all

Reply:
This should now be in the second commit.

As far as backporting goes... we probably should, but I'm not certain if it's realistic.